lib/ansible/modules/system/systemd.py

```diff
@@ -269,7 +269,11 @@
 
 
 def is_running_service(service_status):
-    return service_status['ActiveState'] in set(['active', 'activating'])
+    return service_status['ActiveState'] in set(['active', 'activating', 'deactivating'])
```

Comment:
This doesn't look right.  If the user requests that the service is started then the service should be running after the task ends.  But with this change, if the service is deactivating when the task runs, then the service could be stopped by the time the task ends (or at some point after the task ends and before a task which needs the service to be running starts).

Reply:
@abadger you're absolutely right. I have no idea how I left `deactivating` in there, when I created a separate function below just to check if it's deactivating. Could you remove `deactivating` from the list, or should I do something about it? I could make a separate PR to fix this in devel.

Reply:
@kustodian Make a separate PR in devel.  Ping me and I'll merge that.  Then you can cherry-pick that commit onto this PR and I'll merge this as well.  Thanks!

Reply:
Oh, and the new PR will need a separate backport to the stable-2.9 branch too since we branched that on Thursday last week.

Reply:
I opened the backport PR against `stable-2.9` (#61786) and added the commit to this PR. Sorry I did not catch this during review. Thank you, as always, @abadger.